components/ILIAS/Authentication/classes/class.ilSessionReminderCheck.php

```diff
@@ -93,11 +95,12 @@ public function handle(): ResponseInterface
             return $this->toJsonResponse($response);
         }
 
-        $expiration_time = (int) $data['expires'];
+        $expiration_time = $data['expires'] ?? null;
```

Comment:
The key does always exists, so the `?? null` has no added value.

```suggestion
        $expiration_time = $data['expires'];
```